docs/static/releasenotes.asciidoc

```diff
@@ -13,6 +15,156 @@ This section summarizes the changes in the following releases:
+* Fixes an issue where runaway matchers inside KV and Grok Filter Plugins could fail to respect configured timeouts https://github.com/elastic/logstash/pull/10978[#10978]
+* Enhanced `GET _node/stats/pipelines` API for Metricbeat monitoring https://github.com/elastic/logstash/pull/10576[#10576]
+* Enhanced `GET /` API to include `workers` and `batch_size` metadata https://github.com/elastic/logstash/pull/10853[#10853]
+* Added Plain codec for Java https://github.com/elastic/logstash/pull/10791[#10791]
+* Added JMS Input Plugin to the list of default plugins https://github.com/elastic/logstash/pull/10865[#10865]
+
+==== Plugins
+
+*Grok Filter*
+
+* Changed timeout handling using the Timeout class, resolving an issue where Logstash would fail to enforce timeouts https://github.com/logstash-plugins/logstash-filter-grok/pull/147[#147]
+
+*Http Filter*
+
+*Jdbc_streaming Filter*
+
+* Fixed formatting in documentation https://github.com/logstash-plugins/logstash-filter-jdbc_streaming/pull/17[#17] and https://github.com/logstash-plugins/logstash-filter-jdbc_streaming/pull/28[#28]
+
+*Json Filter*
+
+* Added better error handling, preventing some classes of malformed inputs from crashing the pipeline.
+
+*Kv Filter*
+
+* Changed timeout handling using the Timeout class, resolving an issue where Logstash would fail to enforce timeouts https://github.com/logstash-plugins/logstash-filter-kv/pull/84[#84]
+
+* Fixed asciidoc formatting in docs
+
+* Resolved potential race condition in pipeline shutdown where the timeout enforcer could be shut down while work was still in-flight, potentially leading to stuck pipelines.
+* Resolved potential race condition in pipeline shutdown where work could be submitted to the timeout enforcer after it had been shutdown, potentially leading to stuck pipelines.
+
+*Memcached Filter*
+
+* Fixed link formatting issues in doc https://github.com/logstash-plugins/logstash-filter-memcached/pull/16[#16]
+
+*Mutate Filter*
+
+*Useragent Filter*
+
+* Added support for OS regular expressions that use backreferences https://github.com/logstash-plugins/logstash-filter-useragent/pull/59[#59]
+
+*Azure_event_hubs Input*
+
+* Added workaround to fix errors when using this plugin with Java 11 https://github.com/logstash-plugins/logstash-input-azure_event_hubs/pull/38[#38]
+
+*Kafka Input*
+
+* Added support for `sasl_jaas_config` setting to allow JAAS config per plugin, rather than per JVM https://github.com/logstash-plugins/logstash-input-kafka/pull/313[#313]
+
+*Jms Input*
+
+* The JMS Input is now a default plugin distributed with Logstash
+
+*Snmp Input*
+
+* Adding oid_path_length config option https://github.com/logstash-plugins/logstash-input-snmp/pull/59[#59]
+* Fixing bug with table support removing index value from OIDs https://github.com/logstash-plugins/logstash-input-snmp/issues/60[#60]
+
```

Comment:
Should this issue be closed?  It looks at least partially resolved with https://github.com/logstash-plugins/logstash-input-snmp/pull/59

Reply:
It _looks_ like it is fixed with #59, but we are leaving the ticket open pending verification from the person who reported it.